plugin/trino-prometheus/src/main/java/io/trino/plugin/prometheus/PrometheusConnectorConfig.java

```diff
@@ -152,6 +155,28 @@ public PrometheusConnectorConfig setPassword(String password)
         return this;
     }
 
+    public Map<String, String> getAdditionalHeaders()
+    {
+        return this.additionalHeaders;
+    }
+
+    @Config("prometheus.http.additional.headers")
+    @ConfigDescription("Additional headers to be sent with the HTTP request to Prometheus")
+    public PrometheusConnectorConfig setAdditionalHeaders(String additionalHeaders)
```

Comment:
You can refer to `ThriftHttpMetastoreConfig#setAdditionalHeaders` method when converting string to map.